tests/NCronJob.Tests/RunAtStartupJobTests.cs

```diff
@@ -109,9 +109,9 @@ public async Task ShouldStartStartupJobsBeforeApplicationIsSpunUp()
         await app.UseNCronJobAsync();
         await RunApp(app);
 
-        storage.Entries.Count.ShouldBe(2);
         storage.Entries[0].ShouldBe("SimpleJob");
         storage.Entries[1].ShouldBe("StartingService");
+        storage.Entries.Count.ShouldBe(2);
```

Comment:
I moved all the count assertions after the checks of each individual entries.
This gives us a better chance at troubleshooting a potentially flaky test.